activemq_xml/metadata.csv

```diff
@@ -1,16 +1,12 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
-activemq.queue.avg_enqueue_time,gauge,10,millisecond,,On average  the amount of time (ms) that messages remained enqueued.,0,activemq,avg enq time
-activemq.queue.consumer_count,gauge,10,,,The number of consumers connected,0,activemq,cnsmr count
-activemq.queue.producer_count,gauge,10,,,The number of producers connected,0,activemq,prdcr count
-activemq.queue.max_enqueue_time,gauge,10,millisecond,,The max  the amount of time (ms) that messages remained enqueued.,0,activemq,max enq time
-activemq.queue.min_enqueue_time,gauge,10,millisecond,,The min  the amount of time (ms) that messages remained enqueued.,0,activemq,min enq time
-activemq.queue.memory_pct,gauge,10,percent,,The percentage of memory currently in use,0,activemq,mem pct
-activemq.queue.size,gauge,10,,,The amount of messages that remained queued.,0,activemq,enq size
-activemq.queue.dequeue_count,gauge,10,,,The amount of messages that remained dequeued.,0,activemq,deq size
-activemq.queue.dispatch_count,gauge,10,,,The amount of messages that have been dispatched.,0,activemq,dsptch cnt
-activemq.queue.enqueue_count,gauge,10,,,The amount of messages that have been enqueued.,0,activemq,enq cnt
-activemq.queue.expired_count,gauge,10,,,The amount of messages that have been expired.,0,activemq,exp cnt
-activemq.queue.in_flight_count,gauge,10,,,The amount of messages that have been in flight.,0,activemq,enq cnt
-activemq.broker.store_pct,gauge,10,percent,,The percentage of store in use.,0,activemq,str pct
-activemq.broker.temp_pct,gauge,10,percent,,The percentage of temporary in use.,0,activemq,temp pct
-activemq.broker.memory_pct,gauge,10,percent,,The percentage of memory in use.,0,activemq,mem pct
+activemq.topic.count,gauge,,,,The number of topics.,0,activemq_xml,topic count
+activemq.topic.consumer_count,gauge,10,,,The number of consumers.,0,activemq_xml,topic cnsmr count
+activemq.topic.dequeue_count,gauge,10,,,The total number of messages sent to the queue since the last restart.,0,activemq_xml,topic deq count
```

Comment:
Same here, we could addd the `message` unit to all the metrics related to number of message